.travis.yml

```diff
@@ -37,5 +37,3 @@ script:
   - if [ $POD_LINT == "YES" ]; then
       pod lib lint;
     fi
-after_success:
```

Comment:
I'm guessing this a product of the merge - but we'll want to ensure this is present. I think we'll also need to ensure the macOS variant is building on CI.

Reply:
I'm getting my branches mixed up. Ignore me.

Reply:
@wmcginty I guess I'm suffering from the same mixup. It looks like this is merging into 'master', which already has the codecov stuff so don't we need to get this fixed?

Reply:
@tylermilner On third viewing, you're right. Those changes were merged into master in commit db315e so these do need to be corrected.

Reply:
done